src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/CountPrisonersService.kt

```diff
@@ -15,7 +15,7 @@ class CountPrisonersService(
    * Returns the number of prisoners on given level at a prison
    */
   suspend fun countOfPrisonersOnLevelInPrison(prisonId: String, levelCode: String): Int {
-    return offenderSearchService.findOffendersAtLocation(prisonId, prisonId)
+    return offenderSearchService.findOffendersAtLocation(prisonId, "")
```

Comment:
This is fine and I prefer it to pass `"{prisonId}-"` so don't feel like you have to change but I do wonder if have an optional parameter for the location (defaulting to `""`) would be slightly nicer on the consumer side of the service

Reply:
yeah `"${prisonId}-"` felt more hacky to me

Reply:
e.g. `offenderSearchService.findOffendersAtLocation("MDI")` conveys better what you're trying to do, e.g. get all prisoners in Moorland. But as I said, up to you.